projects/admin/src/app/record/detail-view/item-detail-view/item-detail-view.component.html

```diff
@@ -24,12 +24,16 @@ <h1>{{ 'Barcode' | translate }} {{ record.metadata.barcode }}</h1>
       <!-- CALL NUMBER -->
       <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" translate>Call number</dt>
       <dd class="col-sm-7 col-md-8 mb-0">
-        <ng-container *ngIf="record.metadata.call_number else no_call_number">
+        <ng-container *ngIf="record.metadata.call_number else no_data">
           {{ record.metadata.call_number }}
         </ng-container>
-        <ng-template #no_call_number>
-          <span class="no-data">({{ 'no data' | translate }})</span>
-        </ng-template>
+      </dd>
+      <!-- SECOND CALL NUMBER -->
+      <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" translate>Second call number</dt>
```

Comment:
Why you don't hide this part if second_call_number isn't defined ?

Reply:
Done